### ISSUE TITLE
I lost categories after updating to 9.1

### ISSUE THREAD

**Issue by qcol** [ActivityWatch/activitywatch#387]
## Describe the bug

I know that categories are saved temporarily in the browser. I did not change the browser, I did not clean the data, but after running version 9.1 my own categories (over a dozen) disappeared. Interestingly, the default categories and one (only one) of my own have returned. The rest of mine have disappeared. I think they were still after 9.0. 
Already on version 8.4 it happened so that in the category settings, mine disappeared and appeared default, but after a few refreshes or closing and reopening the page with activity - own categories returned. They don't want to come back this time. Is there any way to recover them? Where are they saved? I would like to be able to make a backup after recreating them.

## To Reproduce
It's hard to say. I didn't find a rule. Once they disappear and come back. This time, they disappeared forever after the 9.1 version was launched.

## Environment
 - ActivityWatch version: 9.1
 - Chrome  79.0.3945.130
 - Ubuntu 18.04.4 LTS

**Comment by ErikBjare** [comment]
Did you use the same URL in the browser window? If you use http://localhost:5600 that will use different categories than http://127.0.0.1:5600 since the browser manages localstorage by host and can't know for sure that they are the same.

**Comment by qcol** [comment]
Actually, it works on localhost:5600  (I checked on 127.0.0.1) I didn't think it could be the cause. It's possible that now I called the site manually because the icon in tray disappeared.   I didn't knowingly use different address myself.  Probably the icon in tray directs to localhost:5600 and one of the links in the web application directs to 127.0.0.1:5600, so sometimes the settings "disappeared"?.  I will try to trace it. Thanks for quick help!
